### DISCUSSION TITLE
Linear Projection widget

### DISCUSSION THREAD

**Discussion by TonciG** [biolab/orange3#6313] [Q&A]
Hi,
Exploring the functionality of this widget and have a couple of questions:
- Is there any documentation or help about Circular Placement? I have an idea what this projection is about but would like to be sure.
- For some reason I cannot select LDA, only PCA and Circular Placement. What could be the reasons for this?
Many thanks,
Tonci

**Comment by janezd** [comment]
1. It just puts variables around the circle. That is, axes are arranged at equal angles. VizRank can change the order to maximize separation between instances of different classes.

2. Hm, what do you mean by LDA, that is, Linear discriminant analysis? Between what does it discriminate? (This is a Socratic question based on my wild guess what's the problem in your case. :)

**Comment by TonciG** [comment]
Hi Janez,
Thank you for your responses.
1. Thought so, just needed confirmation.
2. Yes, I mean Linear Discriminant Analysis (LDA), which is one of the three linear projection techniques supported by the widget. Without venturing into philosophy (no matter how much I like it :-) ), thought that supplying the target variable would allow me to use LDA because this would give it context to discriminate the data. But even when I specify the target variable, LDA is still unavailable, so I can only use either Circular Placement or PCA. So the question is why?
Regards,
Tonci

**Reply by janezd** [reply]
Oops, I'm sorry. I see the problem now: LDA in two dimensions requires at least three different categories. You have just two, I suppose?

The widget should show this as a warning when it disables the option. I'm transferring this to issues. Thanks for this question!

**Reply by TonciG** [reply]
Yes, the target variable is at two levels.
Thanks,
Tonci
